examples/migrations/crewai_to_okf/README.md

```
@@ -0,0 +1,86 @@
+python3 ./examples/migrations/crewai_to_okf/migrate_crewai.py \
+  --source ./examples/migrations/crewai_to_okf/sample_data.json \
+  --output ./examples/migrations/crewai_to_okf/sample_output/okf
+```
+
+---
+
+## 📂 Output Bundle Structure
+
+After migration, the output directory contains:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)